src/custom-elements/src/components/dropdown/dropdown.tsx

```diff
@@ -43,6 +48,10 @@ export class Dropdown {
    * To provide selector of an external reference element
    */
   @Prop() reference: string;
+  /**
+   * To provide style for dropdown menu
+   */
+  @Prop() menuStyle?: string;
```

Comment:
@soujanreddy same here for this prop.

Reply:
@jllr You mean to update the name of this prop?